src/components/FilterPanel/ZipCodeFilter.js

```diff
@@ -2,12 +2,29 @@ import FormControl from "@material-ui/core/FormControl";
 import FormGroup from "@material-ui/core/FormGroup";
 import FormLabel from "@material-ui/core/FormLabel";
 import TextField from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+    error: {
+        "& .MuiFormHelperText-contained.Mui-error": {
+            color: "#AD003B",
+        },
+        "& .Mui-error .MuiOutlinedInput-notchedOutline": {
+            "border-color": "#AD003B",
+        },
+    },
+    label: {
+        "margin-bottom": "5px",
+    },
+});
 
 // For performance, use a pared down list of Mass. zipcodes only (saves 374K or 60% of size!)
 // const zipcodeData = require("us-zips");
 import zipcodeData from "../../generated/ma-zips.json";
```

Comment:
Maybe you need to install **[eslint](https://github.com/livgust/macovidvaccines.com#eslint)**?

Reply:
I have eslint installed, weird that I wasn't getting that error, maybe because I wasn't the one who added the line? Would you mind sending me your eslint config file for comparison?

Reply:
I don't have a config file.  It's just the default installation.  My eslint version is v7.20.0

Reply:
I continue to see this error in the current code on this PR.
You didn't add the line, but in d76e25a5 you added 17 lines prior to it, causing the error.

How are you testing this code — are you using something other than `npm run start` or `npm start`? It should be running eslint automatically. 

(That said, I don't understand how `npm start` does its eslint check, it's different than what `npm run check:eslint` does, the latter produces no error. It is also different from running `eslint` by hand from the command line, which produces many more errors that seem unrelated.)

Reply:
This appears to have been fixed in `RadiusFilter.js` but not in this file.

Reply:
Yes I just use npm run start. Not sure what is up with my configuration. I get warnings for other things (from other files I didn't touch) but not that one specifically

Reply:
Do you have any of 

    .eslintrc.js
    .eslintrc.cjs
    .eslintrc.yaml
    .eslintrc.yml
    .eslintrc.json

in your tree?